modules/apps/frontend-theme/frontend-theme-classic/src/css/_custom.scss

```diff
@@ -1,105 +1,158 @@
-			margin-bottom: 0;
-		}
+	.text-default {
+		align-items: center;
+		display: flex;
 
-		.portlet {
-			margin-bottom: 0;
+		.user-full-name {
+			margin-left: 0.5rem;
 
-			&:hover {
-				z-index: 1034;
+			@include media-breakpoint-down(xs) {
+				display: none;
 			}
 		}
+	}
+}
 
-		.search-input {
-			font-size: 0.88rem;
-			padding: 0.28rem 2rem 0.28rem 1rem;
-		}
+/*
+* Move to Styled then to Portlet module (it doesn't exist yet)
+*/
+.portlet .container-fluid-max-xl {
+	padding-left: 0;
+	padding-right: 0;
+}
+
+.portlet-topper {
+	align-items: center;
+	padding: 0;
 
-		.text-default {
-			color: #6b6c7e;
-			font-weight: 600;
+	// Need the .portlet parent for specifity of Styled
+	.portlet & {
+		@include media-breakpoint-up(sm) {
+			right: auto;
 		}
+	}
 
-		.user-personal-bar {
-			.fieldset {
-				position: relative;
-			}
+	.portlet-title-default {
+		line-height: initial;
+		padding-left: 0.5rem;
+		padding-right: 0.5rem;
+	}
 
-			.user-avatar-link .panel-notifications-count {
-				right: calc(100% - 0.75rem);
-				top: -0.35rem;
-			}
+	// This design requirement is creating an accessibility issue
+	.icon-monospaced {
+		height: 1.5rem;
+		line-height: 1.5rem;
+		width: 1.5rem;
+
+		.lexicon-icon {
+			height: 0.875rem;
+			width: 0.875rem;
 		}
 	}
 
-	.user-avatar-link {
-		display: block;
-		position: relative;
-
-		.panel-notifications-count {
-			display: block;
-			margin: 0;
-			max-width: none;
-			position: absolute;
-			right: calc(100% - 0.7rem);
-			top: -0.4rem;
-		}
+	.controls-visible .portlet:hover &,
+	.controls-visible .portlet.open &,
+	.controls-visible .portlet.focus & {
+		z-index: $zindex-fixed + 4; // 1034
+	}
+}
 
-		.text-default {
-			align-items: center;
-			display: flex;
+/*
+* If title without .portlet-layout doesn't exist
+* Move to Styled then to Portlet module (it doesn't exist yet)
+*/
+.portlet-title-text {
+	color: $secondary;
+	font-size: 0.875rem;
+	font-weight: 600;
+	line-height: 2.2;
+	margin-bottom: 0;
+	text-transform: uppercase;
+}
 
-			.user-full-name {
-				margin-left: 0.5rem;
+/*
+* Move to Fragments?
+*/
+.portlet-dropzone {
+	box-shadow: 0 0 0 1px transparent inset;
+	transition: box-shadow 300ms ease-in-out;
+	will-change: box-shadow;
 
-				@include media-breakpoint-down(xs) {
-					display: none;
-				}
-			}
-		}
+	.yui3-dd-drop-active-valid & {
+		box-shadow: 0 0 0 1px rgba($primary-d2, 0.2) inset;
 	}
-}
 
-.btn-group + .info-panel {
-	@include media-breakpoint-up(sm) {
-		top: auto;
+	.yui3-dd-drop-over & {
+		box-shadow: 0 0 0 4px $primary-d2 inset;
 	}
 }
 
-#footer {
-	background-color: $complementary-color;
-	color: #fff;
-	margin-top: 1rem;
+/*
+* Move to Hello-World module
+* Remember the export the image too
+* Use property class not the portlet's one
+*/
+.portlet-hello-world {
+	.portlet-content {
+		background: #6f5245 url(../images/portlet_hello_world_bg.jpg) bottom
+			left;
+		background-clip: padding-box;
+		background-size: cover;
+		color: $white;
```

Comment:
Consider adding a shadow for accessibility

Reply:
Experimenting with a background contrast

![exp1](https://user-images.githubusercontent.com/46778114/107634716-45e68300-6c6a-11eb-8a7a-2a384ded76b4.jpg)
![exp2](https://user-images.githubusercontent.com/46778114/107634793-644c7e80-6c6a-11eb-9f15-438bf144483b.jpg)

Reply:
That gradient with opacity will be difficult to render, considering that there doesn't seem to be an accessibility issue related to that contrast, I'd rather not add the gradient.
Also, that image is a Design responsibility. :-P

Reply:
The original image is

![image](https://user-images.githubusercontent.com/46778114/109178079-c76d0380-7788-11eb-8fa5-d78b23466bca.png)

and I don't love the legibility of  "Welcome.."

But you are right, I'll leave it to the design team